docs/b2b-edition/specs/storefront/storefront/order.yaml

```diff
@@ -3,8 +3,14 @@ info:
   title: Order
   version: '1.0'
   description: Company based order management.
+  contact:
+    name: BigCommerce
+    url: 'https://www.bigcommerce.com'
+    email: support@bigcommerce.com
 servers:
   - url: 'https://api-b2b.bigcommerce.com/api/v2'
+security:
+  - authToken: []
```

Comment:
For all B2B storefront API's the recommended way to authenticate is `Authorization: Bearer {B2B_JWT_TOKEN}`